UI/Web/src/app/manga-reader/_components/manga-reader/manga-reader.component.ts

```diff
@@ -549,9 +554,33 @@ export class MangaReaderComponent implements OnInit, AfterViewInit, OnDestroy {
         takeUntilDestroyed(this.destroyRef)
       ).subscribe(() => {});
 
+      this.generalSettingsForm.get('pageSplitOption')?.valueChanges.pipe(takeUntilDestroyed(this.destroyRef)).subscribe(val => {
```

Comment:
Let's move the subscribe into a tap so that subscribe can be empty. 

In addition, move each pipe into a new line, like you see elsewhere. 

I would also get the control in a variable then re-use in the code, so:
const widthOverrideControl = this.generalSettingsForm.get('widthSlider')!;